ci/build.yml

```diff
@@ -148,6 +150,7 @@ jobs:
     - template: report-start.yml
     - checkout: self
     - bash: ci/dev-env-install.sh
+    - temaplte: clean-up.yml
```

Comment:
Typo.

Reply:
Thanks! They were all wrong. I guess I should double-check before copy/pasting. All fixed up now. I think.